src/core/entities/MRTextFieldEntity.js

```diff
@@ -16,55 +14,91 @@ export class MRTextFieldEntity extends MRTextInputEntity {
      */
     constructor() {
         super();
-        this.wrapper = this.shadowRoot.appendChild(document.createElement('div'));
-        this.wrapper.innerHTML = '<slot></slot>';
+        this.lineHeight = 1.2; // Default line height, can be adjusted as needed
+        this.object3D.name = 'textField';
+
+        // this.wrapper = this.shadowRoot.appendChild(document.createElement('div'));
```

Comment:
Related to my other comment. why a hidden field in the DOM instead of an actual field in the shadow?